soh/soh/OTRGlobals.cpp

```diff
@@ -2254,7 +2254,7 @@ extern "C" int CustomMessage_RetrieveIfExists(PlayState* play) {
             messageEntry = OTRGlobals::Instance->gRandomizer->GetGoronMessage(choice);
         }
         else if (textId == TEXT_FROGS_UNDERWATER && ctx->GetOption(RSK_FROGS_HINT)) {
-           messageEntry = ctx->GetHint(RH_FROGS_HINT)->GetHintMessage(MF_AUTO_FORMAT), TEXTBOX_TYPE_BLUE;
+           messageEntry = ctx->GetHint(RH_FROGS_HINT)->GetHintMessage(MF_AUTO_FORMAT);
```

Comment:
Did you change this or is there a weird merge conflict resolution thing going on here?

Reply:
I changed this. It's a nop, C has a `,` operator where `(a,b) == b`, which is why this compiled still

Reply:
OK, just making sure. Seemed a bit out of place. Shame it got left in and fixed in an unrelated PR but I guess I'd rather not try to split it off for just a one line fix.

Reply:
I'm with leggett here, please keep unrelated fixes to separate PRs, even if it's as small as this. It makes reviewing PRs with unrelated changes difficult.